tensorflow_serving/g3doc/serving_advanced.md

```diff
@@ -143,8 +143,11 @@ With all these, `ServerCore` internally does the following:
   the `SessionBundleSourceAdapter`.
 
 Whenever a new version is available, this `AspiredVersionsManager` always
-unloads the old version and replaces it with the new one. If you want to
-start customizing, you are encouraged to understand the components that it
+unloads the old version and replaces it with the new one. The default behavior means that
```

Comment:
Let's replace the first sentence with:

Whenever a new version is available, this `AspiredVersionsManager` loads the new version,
and, under its default behavior, unloads the old one.